content/en/developers/ide_integrations/idea/_index.md

```diff
@@ -0,0 +1,155 @@
+1. If you receive a prompt notifying you that Datadog is a third-party plugin, click **Accept**.
+1. Click **Restart IDE** to restart IDEA.
+
+{{< img src="/developers/ide_integrations/idea/datadog-plugin1.png" alt="The Datadog plugin" style="width:100%;" >}}
+
+### Add Datadog credentials
+
+After installing the Datadog plugin and restarting IDEA, add your Datadog API and application keys to authenticate with Datadog:
+1. With a file or project open in IDEA, click the **Datadog** tool window.
+1. Click **Add your credentials...**.
+1. Enter your name, [API key and application key][5], and select a site.
+
+**Note**: For most users, one pair of keys is all that is required. If you're using a multi-org setup with multiple key pairs, check to ensure that the correct pair is active using the **Tools** -> **Datadog** section in **Preferences**.
+
+### Link a service
+
+To provide relevant data from the Datadog platform, add relevant services to a project:
+1. With your project open in IDEA, open the **Datadog** tool window and click the plus icon (**+**).
+1. Search for and select the services that you want to add to the current project.
+
+To remove a service, select it in the **Services** table and click the minus icon (**-**).
+
+<div class="alert alert-info">The names of linked services persist with the project when you close it.</div>
+
+## Using the plugin
+
+After you add a service to your project, right-click on the service and click **Open in Profiling** to open a Profiling tab for the service. A profiling tab displays data for only one service, but you can have multiple tabs open simultaneously.
+
+The Profiling tab shows Continuous Profiling information for the service in a selected environment, aggregated over a specific time frame. Available views are:
+- [Top list](#top-list): Displays a list of the most resource intensive methods for the current profiling measure.
+- [Flame graph](#flame-graph): A flame graph representing stack traces in the profiles.
+
+{{< img src="/developers/ide_integrations/idea/profiling-example1.png" alt="An example of the Profiling tab for a service named product-recommendation" style="width:100%;" >}}
+
+You can specify the following parameters for the profiling data:
+- The profile type to be displayed
+- The environment in which the service is running
+- The time frame for the profile samples to be aggregated
+
+The available profiling types usually include options like **CPU Time** and **Allocated Memory**, but are determined by the platform and vary by language.
+
+## Top list
+
+The **Top List** sub-tab shows the methods that consume the most resources based on the aggregated profile data loaded from the Datadog servers. **Top List** is designed to show a summary of the methods that are most likely to be interesting from a resource consumption point of view.
+
+{{< img src="/developers/ide_integrations/idea/top-list.png" alt="The Top-list view" style="width:100%;" >}}
```

Comment:
This screenshot shows the now-removed `Code Sync` button. An updated version is here: https://user-images.githubusercontent.com/1835893/195761932-d11edc25-bc55-45d9-92ff-d0c1a1efeb62.png